src/CBLQuery_Internal.hh

```diff
@@ -214,18 +203,23 @@ namespace cbl_internal {
         Retained<CBLResultSet> resultSet() {
             return new CBLResultSet(_query, _c4obs->getEnumerator(false));
         }
+        
+        // CBLListenerToken :
+        
+        void willRemove() override                              {setEnabled(false);}
 
         // CBLStoppable :
-
-        void stop() override {
-            setEnabled(false);
-        }
+        
+        void stopStoppable() override                           {setEnabled(false);}
+        void retainStoppable() override                         {retain(this);}
```

Comment:
This looks fishy.  Why make this virtual if all the overrides are doing the same thing?